learned.lic

```diff
@@ -4,13 +4,15 @@
 
 custom_require.call(%w(drinfomon))
 
+learning_time = ((Time.now - DRSkill.start_time) / 60.0 / 60).round(1)
+
 DRSkill.list.select { |item| item.current - item.baseline > 0 }.sort_by { |item| item.current - item.baseline }.reverse.each_slice(4) do |skills|
   output = ''
-  skills.each { |skill| output += skill.name.center(18) + DRSkill.gained_exp(skill.name).to_s.center(6) }
+  skills.each { |skill| output += skill.name.center(18) + DRSkill.gained_exp(skill.name).to_s.center(6) + "(#{(DRSkill.gained_exp(skill.name)/learning_time).round(2)})".center(6)}
```

Comment:
It my be time create a helper method like

```
def format_skill_data(skill, time)
  gain = DRSkill.gained_exp(skill.name)
  skill.name.center(18) + gain.to_s.center(6) + "(#{(gain/time).round(2)})".center(6)
end
```

We could then do something like
```
output = skills.map { |skill| format_skill_data(skill, learning_time)}.join
respond(output)
```